dev-infrastructure/docs/development-setup.md

```diff
@@ -353,8 +355,44 @@ Then register it with the Maestro Server
     ```
     az network vnet subnet update -g <resource-group> -n <subnet-name> --vnet-name <vnet-name> --network-security-group <nsg-name>
     ```
+  - Generate a random alphanumeric string used as a suffix for the User-Assigned Managed Identities of the operators of the cluster
+    > NOTE: The random suffix used has to be different for each cluster to be created
+    ```
+    export OPERATORS_UAMIS_SUFFIX=$(openssl rand -hex 3)
```

Comment:
I wonder if we should use the future clusters name as the suffix instead of a random string. Easier for traceability. Wdyt?

Reply:
The reason I added randomness is because:
* Two different people could create a cluster with the same name
* The same person could create two clusters with the same name, in different RGs
* In the Azure portal UI you see the name of the managed identity

Reply:
+1 on adding some element that could enable us to associate the MIs to the cluster being created.

Reply:
I updated the MIs creation to also include the desired CS Cluster name.

I am not super convinced about it because:
* Different people or even the same person can have the same cluster name for different clusters, which doesn't really help to differentiate then
* Name length limit concerns. If we do this here people will tend to try do to the same or similar when writing automated tests. In those cases reaching the length is quite more probable because names formed there concatenating several parts adding quickly. MIs have a max length of 128 characters. But right now in this name we already have: username (unbounded), operator name (unbounded), cluster name (54 characters), other extra characters like hyphens, ...

Reply:
Thank you Miguel. I really expect authors of the automated tests to take name limits onto account and not copy and paste the command as it or its equivalent.